src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/domain/Vilkår.kt

```diff
@@ -103,9 +103,10 @@ enum class VilkårType(val beskrivelse: String, val gjelderStønader: List<Støn
     EKSEMPEL2("Eksempel 2", listOf(Stønadstype.BARNETILSYN)),
     MÅLGRUPPE("Målgruppe", listOf(Stønadstype.BARNETILSYN)),
     AKTIVITET("Aktivitet", listOf(Stønadstype.BARNETILSYN)),
+    PASSBARN("PassBarn", listOf(Stønadstype.BARNETILSYN)),
```

Comment:
Kan vi enten ha
```suggestion
    PASS_BARN("PassBarn", listOf(Stønadstype.BARNETILSYN)),
```
eller
```suggestion
    PASSBARN("Passbarn", listOf(Stønadstype.BARNETILSYN)),
```
?

Reply:
I såfall `PASS_BARN`, men usikker på dette navnet generelt 😢